beacon_node/network/src/sync/range_sync/chain_collection.rs

```diff
@@ -313,7 +313,7 @@ impl<T: BeaconChainTypes> ChainCollection<T> {
             peer_id,
             beacon_processor_send,
             self.beacon_chain.clone(),
-            self.log.clone(),
+            self.log.new(o!("chain" => chain_id)),
```

Comment:
Hey @AgeManning this is why I removed the chain ID from the logs

Reply:
Oh. I missed this. Nice!